version: "2.1"
services:
  qbittorrent-gluetun-internal:
    image: "\
      linuxserver/qbittorrent@sha256:\
      94e1c4fb34b38e9b529867aefc1185a10710ea4868dcb1c4f395780d1a4c297b\
    "  # 5.0.3
    container_name: qbittorrent-gluetun-internal
    network_mode: service:qbittorrent-gluetun
    env_file:
      - .env-files/linuxserverio.env
    volumes:
      - .volumes/config:/config
      - ${MEDIA_VOLUME}/downloads:/downloads
    depends_on:
      - qbittorrent-gluetun
    restart: on-failure

  qbittorrent-gluetun:
    image: "\
      qmcgaw/gluetun@sha256:\
      2b42bfa046757145a5155acece417b65b4443c8033fb88661a8e9dcf7fda5a00\
    "  # v3.40.0
    container_name: qbittorrent-gluetun
    cap_add:
      - NET_ADMIN
    env_file:
      - .env-files/gluetun.env
    environment:
      # Allow access to host local network
      - FIREWALL_OUTBOUND_SUBNETS=${HOST_NETWORK_SUBNET}
      # Expose health service externally
      - HEALTH_SERVER_ADDRESS=0.0.0.0:9999
    volumes:
      - .volumes/gluetun:/gluetun
    restart: on-failure

networks:
  default:
    name: ${SWAG_NETWORK}
    external: true
